### ISSUE TITLE
Data render issue

### ISSUE THREAD

**Issue by BlitZSenpai** [TypeCellOS/BlockNote#453]
**Describe the bug**
i stored the data typed in the description as you can see in my first picture in my database(mongodb) and when i try to display the data it is displayed in this format as shown in the second picture. the third picture is my 

![Screenshot 2023-12-05 170727](https://github.com/TypeCellOS/BlockNote/assets/122906171/7658779f-37a6-42ad-800e-a86f0034e1ab)
![Screenshot 2023-12-05 170749](https://github.com/TypeCellOS/BlockNote/assets/122906171/edba6b61-dfe8-4a65-8fe7-b83c6c59316d)
![image_2023-12-05_171225381](https://github.com/TypeCellOS/BlockNote/assets/122906171/17812db6-a763-4133-80de-69d5235675bd)
![image_2023-12-05_171248583](https://github.com/TypeCellOS/BlockNote/assets/122906171/eabb782b-0b8a-4cc2-9cd1-97e0ce6d9690)

**Comment by matthewlipski** [comment]
Hmm not sure I entirely understand the issue you're having. `blocknote.topLevelBlocks` gets all the non-nested blocks in the editor in a JSON format - I suppose for your form you just want the text content, is that correct?

**Comment by BlitZSenpai** [comment]
@matthewlipski yeah . i just want the text content to be displayed on the fetched post data. my website is basically a blog website consisting of a title, summary and a description. This descrption has the editor component in it

**Comment by BlitZSenpai** [comment]
@matthewlipski thank you . ill look into the docs and btw by the text content i meant like the markup and other features the blocknote editor gives us. i want the description to be highly customizable so that the user can express his blog post however they want to.

**Comment by YousefED** [comment]
Closing, we'll need a bit clearer description / requirements if this is still an issue in 0.12
